dwave/cloud/client.py

```diff
@@ -737,9 +737,9 @@ def get_solver(self, name=None, features=None, refresh=False):
 
             features (dict, optional):
                 Dictionary of features this solver has to have. For a list of
-                feature names and values, see: :meth:`Client.solvers`. Specifying solver
-                name overrides features. To include (possibly partial) name match requirement,
-                use the ``name`` feature inside ``features`` dictionary.
+                feature names and values, see: :meth:`~dwave.cloud.client.Client.solvers`.
+                Specifying solver name overrides features. To include (possibly partial)
```

Comment:
Specifying the 'name' parameter overrides the 'feature' parameter.

Reply:
Thanks, will change in the next PR.